projects/plugins/jetpack/modules/masterbar/admin-menu/class-base-admin-menu.php

```diff
@@ -656,6 +657,17 @@ public function handle_preferred_view() {
 		// phpcs:enable WordPress.Security.NonceVerification
 	}
 
+	/**
+	 * Adds the necessary CSS class to the admin body class.
+	 *
+	 * @param string $admin_body_classes Contains all the admin body classes.
+	 *
+	 * @return string
+	 */
+	public function admin_body_class( $admin_body_classes ) {
+		return "nav-unification-enabled $admin_body_classes";
```

Comment:
Can you add a space in the beginning and at the end of the string to avoid possible plugin conflicts?

Reply:
Yes, sure thing.

Reply:
In other similar cases, I've seen it done the other way around, which would automagically prevent plugin conflicts.
`"$admin_body_classes nav-unification-enabled"`

Reply:
I've tried like that at first @Copons but I was getting a conflict right off the bat. The nav-unification-enabled class was getting appended like this:
````

nav-unification-enabledfont-smoothing-antialiased
````
I think the only way we can 100% ensure that we won't be getting a conflict is by adding spaces at the beginning and end.